test/uk/gov/hmrc/hec/connectors/SDESConnectorSpec.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hec.connectors
+
+import com.typesafe.config.ConfigFactory
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.Configuration
+import uk.gov.hmrc.hec.models.sdes.{FileAudit, FileChecksum, FileMetaData, SDESFileNotifyRequest}
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+
+import java.util.UUID
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class SDESConnectorSpec extends AnyWordSpec with Matchers with MockFactory with HttpSupport with ConnectorSpec {
+  val (protocol, host, port) = ("http", "host", "123")
+  val config                 = Configuration(
+    ConfigFactory.parseString(s"""
+                                 | microservice.services.sdes {
+                                 |    host     = $host
+                                 |    port     = $port
+                                 |  }
+                                 |  file-notify-config {
+                                 |    file-notify-to = "sdes-stub"
+                                 |}
+                                 |
+                                 |""".stripMargin)
+  )
+
+  val connector     = new SDESConnectorImpl(mockHttp, new ServicesConfig(config), config)
+  val uuid          = UUID.randomUUID()
+  val notifyRequest = SDESFileNotifyRequest(
+    "ssa",
+    FileMetaData(
+      "hec",
+      "file1.dat",
+      s"http://localhost:8464/object-store/object/hec/tax-checks/file1.dat",
```

Comment:
there doesn't seem to be string interpolation here

Reply:
i was planning to pass on the file name, then dropped the idea :), will make it simple string